### PR TITLE
Updated spanish language

### PR DESCRIPTION
**Pull request by FrannDzs**
Added spanish variations (es_ar/es_cl/es_uy/es_ve/es_ec/es_mx)

### CODE REVIEW

**Comment by vectorwing** [comment]
Nice! I'll merge it on the upcoming 0.4.4 release.

This reminds me... I need to properly update the wiki's main page with the status of each translation again, and maybe write a decent doc to help new translators out (especially with GitHub complexities).

File: src/main/resources/assets/farmersdelight/lang/es_es.json [hunk 1]
```diff
@@ -62,30 +67,32 @@
...
-  "item.farmersdelight.pumpkin_slice": "Pedazo de calabaza",
-  "item.farmersdelight.cabbage_leaf": "Hoja de repollo",
-  "item.farmersdelight.minced_beef": "Carne molida",
-  "item.farmersdelight.beef_patty": "Bife de hamburguesa",
-  "item.farmersdelight.chicken_cuts": "Pedazos de pollo crudo",
-  "item.farmersdelight.cooked_chicken_cuts": "Pedazos de pollo cocido",
+  "item.farmersdelight.pumpkin_slice": "Rodaja de calabaza",
+  "item.farmersdelight.cabbage_leaf": "Hoja de col",
+  "item.farmersdelight.minced_beef": "Carne picada",
+  "item.farmersdelight.beef_patty": "Hamburguesa cruda",
```
Comment (vectorwing) [inline src/main/resources/assets/farmersdelight/lang/es_es.json]: Beef Patties are actually the cooked form; the term "cruda" (raw) might be wrong here.

**Review by vectorwing — COMMENTED** [review]
Just a quick detail I spotted, if you'd like to take a look before merging.

File: src/main/resources/assets/farmersdelight/lang/es_es.json (hunk 1 above)
Comment (FrannDzs) [reply src/main/resources/assets/farmersdelight/lang/es_es.json]: sorry :7

Comment (FrannDzs) [reply src/main/resources/assets/farmersdelight/lang/es_es.json]: @vectorwing fixed

**Pull request merged**